roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -156,6 +156,9 @@ networkConfig:
 {% endif %}
 # serviceNetworkCIDR must match kubernetesMasterConfig.servicesSubnet
   serviceNetworkCIDR: {{ openshift.common.portal_net }}
+{% if 'external_ip_network_cidrs' in openshift.master and isinstance(openshift.master.external_ip_network_cidrs, list) and openshift.master.external_ip_network_cidrs | count > 0 %}
+  externalIPNetworkCIDRs: "{{ openshift_master_external_ip_network_cidr | to_padded_yaml(1,2) }}"
+{% endif %}
```

Comment:
lets remove all the complexity here...  remove the fact setting, drop all the conditionals, and lets do something like this:
```
  externalIPNetworkCIDRs: "{{ openshift_master_external_ip_network_cidrs | default(["0.0.0.0/0"]) | to_padded_yaml(1,2) }}"
```